test/e2e/apimachinery/custom_resource_definition.go

```diff
@@ -251,6 +257,128 @@ var _ = SIGDescribe("CustomResourceDefinition resources [Privileged:ClusterAdmin
 		}
 	})
 
+	/*
+		Release : v1.17
+		Testname: Custom Resource Definition, defaulting
+		Description: Create a custom resource definition without default. Create CR. Add default and read CR until
+		the default is applied. Create another CR. Remove default, add default for another field and read CR until
+		new field is defaulted, but old default stays.
+	*/
+	framework.ConformanceIt("custom resource defaulting for requests and from storage works ", func() {
```

Comment:
If this is a new test it needs to be added as an e2e and soak for a bit before we promote it (can't be flaky). are there any existing e2e tests for this?

Reply:
I would suggest a separate PR that adds this as an e2e test that uses the beta API. That way we get a signal on flakiness. then the PR to promote to GA would only promote, not implement that test